--- origsrc/aqbanking-5.6.10/aqbanking-config.cmake.in	2016-01-15 14:15:22.000000000 -0600
+++ src/aqbanking-5.6.10/aqbanking-config.cmake.in	2016-08-28 23:17:01.389628500 -0500
@@ -22,7 +22,7 @@ find_dependency(gwenhywfar "@GWENHYWFAR_
 set_and_check(prefix "@prefix@")
 set_and_check(exec_prefix "@exec_prefix@")
 set_and_check(AQBANKING_INCLUDE_DIRS "@includedir@/aqbanking@AQBANKING_VERSION_MAJOR@")
-set_and_check(AQBANKING_LIBRARIES "@libdir@/libaqbanking.so")
+set_and_check(AQBANKING_LIBRARIES "@libdir@/libaqbanking.dll.a")
 set(AQBANKING_FOUND true)
 
 # The target is available since aqbanking 5.6.5
--- origsrc/aqbanking-5.5.1/configure.ac	2014-12-31 07:25:47.000000000 -0600
+++ src/aqbanking-5.5.1/configure.ac	2015-10-16 13:07:28.093649600 -0500
@@ -1336,12 +1336,15 @@ AC_SUBST(aqbanking_backends)
 
 
 # Adjust CFLAGS and LDFLAGS on Windows
-case "$OS_TYPE" in
+case "$OSYSTEM" in
     windows)
 	# GnuCash compiles with this; maybe we need this, too?
 	#CFLAGS="${CFLAGS} -mms-bitfields"
 	LDFLAGS="${LDFLAGS} -no-undefined -mwindows"
     ;;
+    cygwin)
+	LDFLAGS="${LDFLAGS} -no-undefined"
+    ;;
 esac
 
 
--- origsrc/aqbanking-5.5.1/m4/os.m4	2014-07-23 15:29:51.000000000 -0500
+++ src/aqbanking-5.5.1/m4/os.m4	2015-10-16 13:07:16.192138300 -0500
@@ -88,6 +88,12 @@ case "$host" in
 	AC_DEFINE(OS_PALMOS,1,[if PalmOS is used])
 	OS_TYPE="palmos"
         ;;
+    *-cygwin*)
+	OSYSTEM="cygwin"
+	AC_DEFINE(OS_CYGWIN,1,[if Cygwin is used])
+	AC_DEFINE(OS_POSIX,1,[if this is a POSIX system])
+	OS_TYPE="posix"
+	;;
     *)
 	AC_MSG_WARN([Sorry, but host $host is not supported.
         Please report if it works anyway. We will assume that your system
