--- origsrc/libaqbanking-3.3.0/Makefile.am	2008-01-09 05:41:12.000000000 -0600
+++ src/libaqbanking-3.3.0/Makefile.am	2008-03-29 23:06:16.937500000 -0500
@@ -15,7 +15,7 @@
 aclocaldir = $(datadir)/aclocal
 dist_aclocal_DATA = aqbanking.m4
 
-ACLOCAL_AMFLAGS = $(ADD_ACLOCAL_FLAGS) -I m4
+ACLOCAL_AMFLAGS = -I m4
 
 iheaderdir=${includedir}/aqbanking
 iheader_HEADERS=\
--- origsrc/libaqbanking-3.3.0/aqbanking-config.in.in	2007-11-22 11:38:27.000000000 -0600
+++ src/libaqbanking-3.3.0/aqbanking-config.in.in	2008-03-29 23:08:22.031250000 -0500
@@ -15,7 +15,7 @@
 	    result="$result @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqbanking_ldflags@ @aqbanking_libs@"
+	    result="$result @aqbanking_ldflags@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--libraries++)
 	    result="$result "
@@ -31,7 +31,7 @@
             result=@with_qbanking@
             ;;
 	--qbanking-libraries)
-	    result="$result @aqbanking_ldflags@ @qbanking_libs@"
+	    result="$result @aqbanking_ldflags@ @qbanking_libs@ @qt3_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--qbanking-plugins)
 	    result="$result @qbanking_plugindir@"
--- origsrc/libaqbanking-3.3.0/aqbanking.pc.in	2007-11-22 11:38:27.000000000 -0600
+++ src/libaqbanking-3.3.0/aqbanking.pc.in	2008-03-29 23:09:40.437500000 -0500
@@ -42,6 +42,7 @@
 Description: A library for online banking functions and financial data import/export
 
 Version: @VERSION@
+Require: gwenhywfar
 Libs: @aqbanking_ldflags@ @aqbanking_libs@
-Cflags: @aqbanking_includes@ @gwenhywfar_includes@
+Cflags: @aqbanking_includes@
 
--- origsrc/libaqbanking-3.3.0/bindings/python/_basetypes.py	2007-11-22 11:38:27.000000000 -0600
+++ src/libaqbanking-3.3.0/bindings/python/_basetypes.py	2008-03-29 23:10:17.765625000 -0500
@@ -14,9 +14,12 @@
 elif os.name == 'mac':
     aqb = cdll.aqbanking
     gwen = cdll.gwenhywfar
+elif sys.platform == 'cygwin':
+    aqb = cdll['cygaqbanking-20.dll']
+    gwen = cdll['cyggwenhywfar-47.dll']
 else:
-    aqb = cdll['libaqbanking.so.16']
-    gwen = cdll['libgwenhywfar.so.38']
+    aqb = cdll['libaqbanking.so.20']
+    gwen = cdll['libgwenhywfar.so.47']
 
 
 _c_char_p = c_char_p
--- origsrc/libaqbanking-3.3.0/configure.ac	2008-03-16 09:32:15.000000000 -0500
+++ src/libaqbanking-3.3.0/configure.ac	2008-03-29 23:29:00.000000000 -0500
@@ -541,7 +541,7 @@
 AC_ARG_WITH(frontends, 
   [  --with-frontends=list      space separated list of frontends (default:all)],
   [aqbanking_frontends="$withval"],
-  [aqbanking_frontends="qbanking"]
+  [aqbanking_frontends="fbanking qbanking"]
 )
 # The ordering is important due to linker dependencies, so force
 # the correct ordering here:
@@ -754,7 +754,7 @@
 #
 
 case "$aqbanking_frontends" in *fbanking*)
-  pkg_modules="fox >= 1.6.0"
+  pkg_modules="fox-1.6 >= 1.6.0"
   PKG_CHECK_MODULES(FOX, [$pkg_modules], [have_fox="yes"], [have_fox="no"])
   if test "$have_fox" = "yes"; then
     AC_DEFINE(HAVE_FOX, 1, [if Fox Toolkit is available])
@@ -991,11 +991,11 @@
 if test "$OS_TYPE" != "windows"; then
   oldlibs="$LIBS"
   LIBS=""
-  AC_SEARCH_LIBS(gettext, intl, [], [HAVE_I18N="no"])
-  i18n_libs="$LIBS"
+  AC_SEARCH_LIBS(gettext, intl, [AM_ICONV], [HAVE_I18N="no"])
+  i18n_libs="$LIBS $LTLIBICONV"
   LIBS="$oldlibs"
 else
-  i18n_libs="-lintl"
+  i18n_libs="-lintl -liconv"
 fi
 
 if test "$HAVE_I18N" != "NO"; then
@@ -1213,8 +1213,8 @@
 
     aqgeldkarte)
       if test "x$have_chipcard_client" != "xyes"; then
-        AC_MSG_NOTICE([*** Libchipcard2-client is required for backend "AqGeldKarte". Specify --with-backends="aqhbci aqdtaus aqofxconnect" to build aqbanking without that backend.])
-	dependencies_ok="no"
+        AC_MSG_NOTICE([*** Libchipcard4-client is required for backend "AqGeldKarte". Specify --with-backends="aqhbci aqdtaus aqofxconnect" to build aqbanking without that backend.])
+        aqbanking_backends="${aqbanking_backends/aqgeldkarte/}"
       fi
       ;;
 
@@ -1245,7 +1245,7 @@
 *** Fox Toolkit is required for FOX-frontend "fbanking". 
 *** Specify --with-frontends="qbanking" to build 
 *** aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/fbanking/}"
       fi
       ;;
   esac
@@ -1528,12 +1528,15 @@
 
 
 # Adjust CFLAGS and LDFLAGS on Windows
-case "$OS_TYPE" in
+case "$OSYSTEM" in
     windows)
 	# GnuCash compiles with this; maybe we need this, too?
 	#CFLAGS="${CFLAGS} -mms-bitfields"
 	LDFLAGS="${LDFLAGS} -no-undefined -mwindows"
     ;;
+    cygwin)
+	LDFLAGS="${LDFLAGS} -no-undefined"
+    ;;
 esac
 
 ###-------------------------------------------------------------------------
--- origsrc/libaqbanking-3.3.0/m4/os.m4	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/m4/os.m4	2008-03-29 23:23:53.312500000 -0500
@@ -47,6 +47,11 @@
 	AC_DEFINE(OS_DARWIN,1,[if Darwin is used])
 	OS_TYPE="posix"
 	;;
+    *-cygwin*)
+	OSYSTEM="cygwin"
+	AC_DEFINE(OS_CYGWIN,1,[if Cygwin is used])
+	OS_TYPE="posix"
+	;;
     *-win32*)
     	OSYSTEM="windows"
 	AC_DEFINE(OS_WIN32,1,[if WIN32 is used])
--- origsrc/libaqbanking-3.3.0/m4/qt3.m4	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/m4/qt3.m4	2008-03-29 23:23:53.328125000 -0500
@@ -149,11 +149,14 @@
 fi
 
 # This is the name of the qt library to search for.
-if test "x$OSYSTEM" = "xdarwin"; then
-   qt_searchname="lib${qt_libname}.3.dylib"
-else
-   qt_searchname="lib${qt_libname}${std_shrext}.3"
-fi
+case "$OSYSTEM" in
+darwin)
+   qt_searchname="lib${qt_libname}.3.dylib" ;;
+cygwin)
+   qt_searchname="lib${qt_libname}.dll.a" ;;
+*)
+   qt_searchname="lib${qt_libname}${std_shrext}.3" ;;
+esac
 
 
 if test -z "$qt3_libs"; then
@@ -235,6 +238,7 @@
 	AC_MSG_RESULT(not found)
 fi
 
+qt3_uic="$qt3_uic -nounload"
 
 # check if all necessary qt3 components where found
 if test -z "$qt3_includes" || \
--- origsrc/libaqbanking-3.3.0/mksymlinks.sh	2008-03-16 09:32:58.000000000 -0500
+++ src/libaqbanking-3.3.0/mksymlinks.sh	2008-03-30 01:33:37.765625000 -0500
@@ -78,7 +78,7 @@
 ln -s "${bprefix}/system.h" headers/aqbanking/system.h
 
 # symlink all headers from src/frontends
-dirlist=' qbanking'  # list might be empty
+dirlist=' qbanking fbanking'  # list might be empty
 for frontend in ${dirlist} ; do
   mkdir -p headers/${frontend}
   symlinkFolder "src/frontends/${frontend}" "headers/${frontend}"
--- origsrc/libaqbanking-3.3.0/src/frontends/fbanking/lib/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/fbanking/lib/Makefile.am	2008-03-30 01:54:50.968750000 -0500
@@ -6,7 +6,7 @@
   -I$(top_builddir)/headers \
   -I$(srcdir)/dialogs -I$(srcdir)/widgets -I$(srcdir)/views \
   -Idialogs -Iwidgets -Iviews \
-  -I$(top_srcdir)/src/libs/aqbanking++ \
+  -I$(top_srcdir)/src/libs \
   $(fox_includes) $(all_includes)
 
 DEFS += -DBUILDING_FBANKING \
--- origsrc/libaqbanking-3.3.0/src/frontends/fbanking/lib/fbanking.pc.in	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/fbanking/lib/fbanking.pc.in	2008-03-29 23:17:59.046875000 -0500
@@ -6,5 +6,6 @@
 Name: fbanking
 Description: FBanking is the AqBanking frontend for the Fox Toolkit
 Version: @FBANKING_VERSION_MAJOR@.@FBANKING_VERSION_MINOR@.@FBANKING_VERSION_PATCHLEVEL@
+Require: aqbanking
 Libs: @fbanking_libs@
 Cflags: @fbanking_includes@
--- origsrc/libaqbanking-3.3.0/src/frontends/qbanking/bin/helpbrowser/Makefile.am	2007-11-22 11:38:26.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/qbanking/bin/helpbrowser/Makefile.am	2008-03-29 23:23:53.328125000 -0500
@@ -47,7 +47,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
--- origsrc/libaqbanking-3.3.0/src/frontends/qbanking/lib/dialogs/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/qbanking/lib/dialogs/Makefile.am	2008-03-29 23:23:53.328125000 -0500
@@ -132,7 +132,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/libaqbanking-3.3.0/src/frontends/qbanking/lib/gui/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/qbanking/lib/gui/Makefile.am	2008-03-30 01:49:20.781250000 -0500
@@ -65,7 +65,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/libaqbanking-3.3.0/src/frontends/qbanking/lib/views/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/qbanking/lib/views/Makefile.am	2008-03-29 23:23:53.343750000 -0500
@@ -52,7 +52,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/libaqbanking-3.3.0/src/frontends/qbanking/lib/widgets/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/frontends/qbanking/lib/widgets/Makefile.am	2008-03-29 23:23:53.343750000 -0500
@@ -63,7 +63,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/libaqbanking-3.3.0/src/libs/Makefile.am	2007-11-22 11:38:27.000000000 -0600
+++ src/libaqbanking-3.3.0/src/libs/Makefile.am	2008-03-30 01:55:51.968750000 -0500
@@ -1,5 +1,5 @@
 DIST_SUBDIRS=aqbanking aqbanking++
-SUBDIRS=aqbanking
+SUBDIRS=aqbanking aqbanking++
 
 sources:
 	for d in $(SUBDIRS); do \
--- origsrc/libaqbanking-3.3.0/src/libs/aqbanking++/Makefile.am	2007-11-22 11:38:26.000000000 -0600
+++ src/libaqbanking-3.3.0/src/libs/aqbanking++/Makefile.am	2008-03-30 01:56:49.812500000 -0500
@@ -1,4 +1,4 @@
-INCLUDES=-I$(top_builddir)/headers $(gwenhywfar_includes)
+INCLUDES=-I$(top_builddir)/headers -I$(top_srcdir)/src/libs $(gwenhywfar_includes)
 
 lib_LTLIBRARIES=libaqbankingpp.la
 
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqdtaus/ui/qt3/libs/Makefile.am	2007-11-22 11:38:22.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqdtaus/ui/qt3/libs/Makefile.am	2008-03-29 23:23:53.421875000 -0500
@@ -15,7 +15,7 @@
  cfgmoduledtaus_p.h
 
 
-aqdtaus_la_LDFLAGS=@STRIPALL@ -module -no-undefined
+aqdtaus_la_LDFLAGS=@STRIPALL@ -module -no-undefined -avoid-version
 
 aqdtaus_la_LIBADD=\
   dialogs/libdialogs.la \
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqdtaus/ui/qt3/libs/dialogs/Makefile.am	2007-11-22 11:38:22.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqdtaus/ui/qt3/libs/dialogs/Makefile.am	2008-03-29 23:23:53.421875000 -0500
@@ -37,7 +37,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqgeldkarte/ui/qt3/libs/Makefile.am	2007-11-22 11:38:23.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqgeldkarte/ui/qt3/libs/Makefile.am	2008-03-29 23:23:53.468750000 -0500
@@ -15,7 +15,7 @@
  cfgmodulegeldkarte_p.h
 
 
-aqgeldkarte_la_LDFLAGS=@STRIPALL@ -module
+aqgeldkarte_la_LDFLAGS=@STRIPALL@ -module -avoid-version
 
 aqgeldkarte_la_LIBADD=\
   dialogs/libdialogs.la \
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqhbci/plugin/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqhbci/plugin/Makefile.am	2008-03-29 23:23:53.468750000 -0500
@@ -84,7 +84,7 @@
 
 testlib_SOURCES=testlib.c
 testlib_LDADD = \
-  $(aqhbci_internal_libs) \
+  libaqhbci.la \
   $(cbanking_internal_libs) \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/Makefile.am	2007-11-22 11:38:24.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/Makefile.am	2008-03-29 23:23:53.500000000 -0500
@@ -62,7 +62,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/wizard/Makefile.am	2007-11-22 11:38:24.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/wizard/Makefile.am	2008-03-29 23:23:53.515625000 -0500
@@ -115,7 +115,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am	2007-11-22 11:38:24.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am	2008-03-29 23:23:53.515625000 -0500
@@ -52,7 +52,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/libaqbanking-3.3.0/src/plugins/backends/aqofxconnect/ui/qt3/libs/Makefile.am	2007-11-22 11:38:25.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/backends/aqofxconnect/ui/qt3/libs/Makefile.am	2008-03-29 23:23:53.546875000 -0500
@@ -33,7 +33,7 @@
  cfgtabpageaccountofx.ui.h \
  cfgtabpageuserofx.ui.h
 
-aqofxconnect_la_LDFLAGS=@STRIPALL@ -module
+aqofxconnect_la_LDFLAGS=@STRIPALL@ -module -avoid-version
 
 aqofxconnect_la_LIBADD=\
   $(aqofxconnect_internal_libs) \
@@ -55,7 +55,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/libaqbanking-3.3.0/src/plugins/bankinfo/at/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/bankinfo/at/Makefile.am	2008-03-29 23:22:24.625000000 -0500
@@ -13,7 +13,7 @@
  at.c
 
 at_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs) 
-at_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+at_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/libaqbanking-3.3.0/src/plugins/bankinfo/ca/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/bankinfo/ca/Makefile.am	2008-03-29 23:22:34.187500000 -0500
@@ -13,7 +13,7 @@
  ca.c
 
 ca_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs) 
-ca_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+ca_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/libaqbanking-3.3.0/src/plugins/bankinfo/ch/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/bankinfo/ch/Makefile.am	2008-03-29 23:22:37.640625000 -0500
@@ -13,7 +13,7 @@
  ch.c
 
 ch_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs) 
-ch_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+ch_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/libaqbanking-3.3.0/src/plugins/bankinfo/de/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/bankinfo/de/Makefile.am	2008-03-29 23:22:40.578125000 -0500
@@ -13,7 +13,7 @@
  de.c
 
 de_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la @ktoblzcheck_libs@ $(gwenhywfar_libs) 
-de_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+de_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/libaqbanking-3.3.0/src/plugins/bankinfo/us/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/bankinfo/us/Makefile.am	2008-03-29 23:22:52.234375000 -0500
@@ -13,7 +13,7 @@
  us.c
 
 us_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs) 
-us_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+us_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/csv/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/csv/Makefile.am	2008-03-29 23:23:53.546875000 -0500
@@ -22,7 +22,7 @@
  csv.c
 
 csv_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-csv_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+csv_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 csv_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/dtaus/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/dtaus/Makefile.am	2008-03-29 23:23:53.578125000 -0500
@@ -19,7 +19,7 @@
  dtaus.c
 
 dtaus_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-dtaus_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+dtaus_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 dtaus_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/eri/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/eri/Makefile.am	2008-03-29 23:23:53.593750000 -0500
@@ -21,7 +21,7 @@
  eri.c
 
 eri_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-eri_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+eri_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 eri_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/eri2/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/eri2/Makefile.am	2008-03-29 23:23:53.593750000 -0500
@@ -22,7 +22,7 @@
  eri2.c
 
 eri2_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-eri2_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+eri2_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 eri2_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/ofx/Makefile.am	2008-01-09 05:29:50.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/ofx/Makefile.am	2008-03-30 02:05:46.109375000 -0500
@@ -8,7 +8,7 @@
   ofx.xml.in
 
 
-INCLUDES = -I$(top_builddir)/headers $(gwenhywfar_includes) -Iparser
+INCLUDES = -I$(top_builddir)/headers $(gwenhywfar_includes) -I$(srcdir)/parser
 #EXTRA_DIST=README
 
 imexporterplugindir = $(aqbanking_plugindir)/imexporters
@@ -27,7 +27,7 @@
   $(aqbanking_internal_libs) $(gwenhywfar_libs) 
 
 ofx_la_LDFLAGS = -no-undefined @STRIPALL@ \
-  -module -omit-version
+  -module -avoid-version
 
 # Add the linker flags for windows resource file
 ofx_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/openhbci1/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/openhbci1/Makefile.am	2008-03-29 23:23:53.625000000 -0500
@@ -19,7 +19,7 @@
  openhbci1.c
 
 openhbci1_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-openhbci1_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+openhbci1_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 openhbci1_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/qif/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/qif/Makefile.am	2008-03-29 23:23:35.062500000 -0500
@@ -19,7 +19,7 @@
  qif.c
 
 qif_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-qif_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+qif_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 qif_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/swift/Makefile.am	2007-11-22 11:38:18.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/swift/Makefile.am	2008-03-29 23:23:53.640625000 -0500
@@ -20,7 +20,7 @@
  swift.c
 
 swift_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-swift_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+swift_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 swift_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/xmldb/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/xmldb/Makefile.am	2008-03-29 23:23:53.671875000 -0500
@@ -20,7 +20,7 @@
  xmldb.c
 
 xmldb_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-xmldb_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+xmldb_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 xmldb_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/imexporters/yellownet/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/imexporters/yellownet/Makefile.am	2008-03-29 23:23:53.671875000 -0500
@@ -19,7 +19,7 @@
  yellownet.c
 
 yellownet_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-yellownet_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+yellownet_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 yellownet_la_DEPENDENCIES = \
--- origsrc/libaqbanking-3.3.0/src/plugins/parsers/dtaus/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/parsers/dtaus/Makefile.am	2008-03-29 23:23:53.687500000 -0500
@@ -28,7 +28,7 @@
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs) 
 
-dtaus_la_LDFLAGS = -no-undefined $(STRIPALL) -module -omit-version
+dtaus_la_LDFLAGS = -no-undefined $(STRIPALL) -module -avoid-version
 
 
 # Add the linker flags for windows resource file
--- origsrc/libaqbanking-3.3.0/src/plugins/parsers/swift/Makefile.am	2007-11-22 11:38:17.000000000 -0600
+++ src/libaqbanking-3.3.0/src/plugins/parsers/swift/Makefile.am	2008-03-29 23:23:53.718750000 -0500
@@ -27,7 +27,7 @@
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs) 
 
-swift_la_LDFLAGS = -no-undefined $(STRIPALL) -module -omit-version
+swift_la_LDFLAGS = -no-undefined $(STRIPALL) -module -avoid-version
 
 
 # Add the linker flags for windows resource file
