--- origsrc/aqbanking-2.3.2/Makefile.am	2007-02-13 12:23:11.000000000 -0600
+++ src/aqbanking-2.3.2/Makefile.am	2007-07-16 07:12:08.671875000 -0500
@@ -14,7 +14,7 @@
 aclocaldir = $(datadir)/aclocal
 dist_aclocal_DATA = aqbanking.m4
 
-ACLOCAL_AMFLAGS = $(ADD_ACLOCAL_FLAGS) -I m4
+ACLOCAL_AMFLAGS = -I m4
 
 iheaderdir=${includedir}/aqbanking
 iheader_HEADERS=\
--- origsrc/aqbanking-2.3.2/aqbanking-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/aqbanking-config.in.in	2007-07-16 07:20:24.015625000 -0500
@@ -15,7 +15,7 @@
 	    result="$result @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqbanking_ldflags@ @aqbanking_libs@"
+	    result="$result @aqbanking_ldflags@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--libraries++)
 	    result="$result "
--- origsrc/aqbanking-2.3.2/aqbanking.pc.in	2006-11-16 14:28:46.000000000 -0600
+++ src/aqbanking-2.3.2/aqbanking.pc.in	2007-07-16 07:21:15.515625000 -0500
@@ -42,6 +42,7 @@
 Description: A library for online banking functions and financial data import/export
 
 Version: @VERSION@
+Require: gwenhywfar
 Libs: @aqbanking_ldflags@ @aqbanking_libs@
-Cflags: @aqbanking_includes@ @gwenhywfar_includes@
+Cflags: @aqbanking_includes@
 
--- origsrc/aqbanking-2.3.2/bindings/python/_basetypes.py	2006-09-21 12:52:09.000000000 -0500
+++ src/aqbanking-2.3.2/bindings/python/_basetypes.py	2007-07-16 07:12:08.718750000 -0500
@@ -14,6 +14,9 @@
 elif os.name == 'mac':
     aqb = cdll.aqbanking
     gwen = cdll.gwenhywfar
+elif sys.platform == 'cygwin':
+    aqb = cdll['cygaqbanking-16.dll']
+    gwen = cdll['cyggwenhywfar-38.dll']
 else:
     aqb = cdll['libaqbanking.so.16']
     gwen = cdll['libgwenhywfar.so.38']
--- origsrc/aqbanking-2.3.2/configure.ac	2007-07-08 05:28:01.000000000 -0500
+++ src/aqbanking-2.3.2/configure.ac	2007-07-16 07:12:08.765625000 -0500
@@ -685,7 +685,7 @@
 AC_ARG_WITH(frontends, 
   [  --with-frontends=list      space separated list of frontends (default:all)],
   [aqbanking_frontends="$withval"],
-  [aqbanking_frontends="cbanking g2banking qbanking kbanking"]
+  [aqbanking_frontends="cbanking fbanking g2banking qbanking kbanking"]
 )
 # The ordering is important due to linker dependencies, so force
 # the correct ordering here:
@@ -985,7 +985,7 @@
 #
 
 case "$aqbanking_frontends" in *fbanking*)
-  pkg_modules="fox >= 1.6.0"
+  pkg_modules="fox-1.6 >= 1.6.0"
   PKG_CHECK_MODULES(FOX, [$pkg_modules], [have_fox="yes"], [have_fox="no"])
   if test "$have_fox" = "yes"; then
     AC_DEFINE(HAVE_FOX, 1, [if Fox Toolkit is available])
@@ -1034,7 +1034,8 @@
 AC_CHECK_PROG(XGETTEXT,xgettext,xgettext, missing)
 if test "$XGETTEXT" != "missing"; then
   AC_CHECK_HEADERS([locale.h libintl.h], [], [HAVE_I18N="no"])
-  AC_SEARCH_LIBS(gettext, intl, [], [HAVE_I18N="no"])
+  AC_SEARCH_LIBS(gettext, intl, [AM_ICONV], [HAVE_I18N="no"])
+  LIBS="$LIBS $LTLIBICONV"
 else
   HAVE_I18N="no"
   AC_MSG_WARN([xgettext is missing. Locale suport is disabled.])
@@ -1304,14 +1305,14 @@
     aqofxconnect)
       if test "x$HAVE_OFX_WITH_CONNECT" != "xyes"; then
         AC_MSG_NOTICE([*** LibOFX>=0.8.0 is required for backend "aqofxconnect". Specify --with-backends="aqhbci aqdtaus aqgeldkarte" to build aqbanking without that backend.])
-	dependencies_ok="no"
+        aqbanking_backends="${aqbanking_backends/aqofxconnect/}"
       fi
       ;;
 
     aqgeldkarte)
       if test "x$have_chipcard_client" != "xyes"; then
-        AC_MSG_NOTICE([*** Libchipcard2-client is required for backend "AqGeldKarte". Specify --with-backends="aqhbci aqdtaus aqofxconnect" to build aqbanking without that backend.])
-	dependencies_ok="no"
+        AC_MSG_NOTICE([*** Libchipcard3-client is required for backend "AqGeldKarte". Specify --with-backends="aqhbci aqdtaus aqofxconnect" to build aqbanking without that backend.])
+        aqbanking_backends="${aqbanking_backends/aqgeldkarte/}"
       fi
       ;;
 
@@ -1330,24 +1331,24 @@
     g2banking)
       if test -z "${GTK2_LIBS}"; then
         AC_MSG_NOTICE([*** GTK2 is required for GTK2-frontend "g2banking". Specify --with-frontends="cbanking qbanking kbanking" to build aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/g2banking/}"
       fi
       if test "x${LIBGLADE2_LIBS}" = "x"; then
         AC_MSG_NOTICE([*** The library "libglade2" is required for GTK2-frontend "g2banking". Specify --with-frontends="cbanking qbanking kbanking" to build aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/g2banking/}"
       fi
       ;;
     qbanking)
       if test "x$have_qt3" != "xyes"; then
         AC_MSG_NOTICE([*** QT3 is required for QT-frontend "qbanking". Specify --with-frontends="cbanking g2banking" to build aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/qbanking/}"
       fi
       qt3_ui="qt3"
       ;;
     kbanking)
       if test "x$have_kde3" != "xyes"; then
         AC_MSG_NOTICE([*** KDE3 is required for KDE-frontend "kbanking". Specify --with-frontends="cbanking g2banking qbanking" to build aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/kbanking/}"
       fi
       case $aqbanking_frontends in 
         *qbanking*)
@@ -1363,7 +1364,7 @@
 *** Fox Toolkit is required for FOX-frontend "fbanking". 
 *** Specify --with-frontends="cbanking g2banking qbanking kbanking" to build 
 *** aqbanking without that frontend.])
-	dependencies_ok="no"
+        aqbanking_frontends="${aqbanking_frontends/fbanking/}"
       fi
       ;;
   esac
@@ -1725,7 +1726,7 @@
     windows)
 	# GnuCash compiles with this; maybe we need this, too?
 	#CFLAGS="${CFLAGS} -mms-bitfields"
-	LDFLAGS="${LDFLAGS} -no-undefined -mwindows"
+	LDFLAGS="${LDFLAGS} -mwindows"
     ;;
 esac
 
--- origsrc/aqbanking-2.3.2/m4/chipcard3.m4	1969-12-31 18:00:00.000000000 -0600
+++ src/aqbanking-2.3.2/m4/chipcard3.m4	2007-07-16 07:12:08.796875000 -0500
@@ -0,0 +1,247 @@
+# $Id: aqbanking-2.3.2-2.src.patch,v 1.1 2007-07-17 07:08:11 yselkowitz Exp $
+# (c) 2004-2006 Martin Preuss<martin@libchipcard.de>
+# This function checks for chipcard3-client and chipcard3-server
+
+AC_DEFUN([AC_CHIPCARD3_CLIENT], [
+dnl searches for chipcard_client
+dnl Arguments: 
+dnl   $1: major version minimum
+dnl   $2: minor version minimum
+dnl   $3: patchlevel version minimum
+dnl   $4: build version minimum
+dnl Returns: chipcard_client_dir
+dnl          chipcard_client_datadir
+dnl          chipcard_client_libs
+dnl          chipcard_client_includes
+dnl          chipcard_client_servicelibs
+dnl          have_chipcard_client
+
+if test -z "$1"; then vma="0"; else vma="$1"; fi
+if test -z "$2"; then vmi="1"; else vmi="$2"; fi
+if test -z "$3"; then vpl="0"; else vpl="$3"; fi
+if test -z "$4"; then vbld="0"; else vbld="$4"; fi
+
+AC_MSG_CHECKING(if chipcard_client support desired)
+AC_ARG_ENABLE(chipcard-client,
+  [  --enable-chipcard-client      enable chipcard_client support (default=yes)],
+  enable_chipcard_client="$enableval",
+  enable_chipcard_client="yes")
+AC_MSG_RESULT($enable_chipcard_client)
+
+have_chipcard_client="no"
+chipcard_client_dir=""
+chipcard_client_datadir=""
+chipcard_client_libs=""
+chipcard_client_infolib=""
+chipcard_client_servicelibs=""
+chipcard_client_includes=""
+chipcard_client_servicedir=""
+if test "$enable_chipcard_client" != "no"; then
+  AC_MSG_CHECKING(for chipcard_client)
+  AC_ARG_WITH(chipcard-client-dir, [  --with-chipcard-client-dir=DIR
+                            uses chipcard_client from given dir],
+    [lcc_dir="$withval"],
+    [lcc_dir="${prefix} \
+	     /usr/local \
+             /usr \
+	     /chipcard-client \
+             /sw \
+             /"])
+
+  for li in $lcc_dir; do
+      if test -x "$li/bin/chipcard3-config"; then
+          chipcard_client_dir="$li";
+          break
+      fi
+  done
+  if test -z "$chipcard_client_dir"; then
+      AC_MSG_RESULT([not found ])
+      AC_MSG_NOTICE([
+*** The library libchipcard3c was not found. Obtain it from 
+*** http://www.libchipcard.de.
+*** If it is already installed (including the -devel package), 
+*** you might need to specify the location with the 
+*** option --with-chipcard-client-dir=DIR.
+***
+*** Please note that it is not Libchipcard this package requested, it is the
+*** successor chipcard3.])
+  else
+      AC_MSG_RESULT($chipcard_client_dir)
+      AC_MSG_CHECKING(for chipcard-client libs)
+      chipcard_client_libs="`$chipcard_client_dir/bin/chipcard3-config --client-libs`"
+      AC_MSG_RESULT($chipcard_client_libs)
+      AC_MSG_CHECKING(for chipcard-client includes)
+      chipcard_client_includes="`$chipcard_client_dir/bin/chipcard3-config --includes`"
+      AC_MSG_RESULT($chipcard_client_includes)
+      AC_MSG_CHECKING(for chipcard-client datadir)
+      chipcard_client_datadir="`$chipcard_client_dir/bin/chipcard3-config --client-datadir`"
+      AC_MSG_RESULT($chipcard_client_datadir)
+      AC_MSG_CHECKING(for chipcard-client servicelib)
+      chipcard_client_servicelibs="`$chipcard_client_dir/bin/chipcard3-config --service-libs`"
+      AC_MSG_RESULT($chipcard_client_servicelibs)
+  fi
+  AC_MSG_CHECKING(if chipcard_client test desired)
+  AC_ARG_ENABLE(chipcard-client-test,
+    [  --enable-chipcard-client-test   enable chipcard_client-test (default=yes)],
+     enable_chipcard_client_test="$enableval",
+     enable_chipcard_client_test="yes")
+  AC_MSG_RESULT($enable_chipcard_client_test)
+  AC_MSG_CHECKING(for Chipcard-Client version >=$vma.$vmi.$vpl.$vbld)
+  if test "$enable_chipcard_client_test" != "no"; then
+    chipcard_client_versionstring="`$chipcard_client_dir/bin/chipcard3-config --vstring`.`$chipcard_client_dir/bin/chipcard3-config --vbuild`"
+    AC_MSG_RESULT([found $chipcard_client_versionstring])
+    if test "$vma" -gt "`$chipcard_client_dir/bin/chipcard3-config --vmajor`"; then
+      AC_MSG_NOTICE([Your Chipcard-Client version is way too old.
+      Please update from http://sf.net/projects/libchipcard])
+    elif test "$vma" = "`$chipcard_client_dir/bin/chipcard3-config --vmajor`"; then
+      if test "$vmi" -gt "`$chipcard_client_dir/bin/chipcard3-config --vminor`"; then
+        AC_MSG_NOTICE([Your Chipcard-Client version is too old.
+          Please update from http://sf.net/projects/libchipcard])
+      elif test "$vmi" = "`$chipcard_client_dir/bin/chipcard3-config --vminor`"; then
+          if test "$vpl" -gt "`$chipcard_client_dir/bin/chipcard3-config --vpatchlevel`"; then
+            AC_MSG_NOTICE([Your Chipcard-Client version is a little bit too old.
+            Please update from http://sf.net/projects/libchipcard])
+          elif test "$vpl" = "`$chipcard_client_dir/bin/chipcard3-config --vpatchlevel`"; then
+            if test "$vbld" -gt "`$chipcard_client_dir/bin/chipcard3-config --vbuild`"; then
+              AC_MSG_NOTICE([Your Chipcard-Client version is a little bit too old. 
+  Please update to the latest CVS version. Instructions for accessing 
+  CVS can be found on http://sf.net/projects/libchipcard])
+             fi
+           fi
+      fi
+    fi
+    have_chipcard_client="yes"
+    #AC_MSG_RESULT(yes)
+  else
+    have_chipcard_client="yes"
+    AC_MSG_RESULT(assuming yes)
+  fi
+dnl end of "if enable-chipcard-client"
+fi
+
+AC_SUBST(chipcard_client_dir)
+AC_SUBST(chipcard_client_datadir)
+AC_SUBST(chipcard_client_servicelibss)
+AC_SUBST(chipcard_client_libs)
+AC_SUBST(chipcard_client_includes)
+])
+
+
+
+
+
+AC_DEFUN([AC_CHIPCARD3_SERVER], [
+dnl searches for chipcard_server
+dnl Arguments: 
+dnl   $1: major version minimum
+dnl   $2: minor version minimum
+dnl   $3: patchlevel version minimum
+dnl   $4: build version minimum
+dnl Returns: chipcard_server_datadir
+dnl          chipcard_server_driverdir
+dnl          chipcard_server_servicedir
+dnl          have_chipcard_server
+
+if test -z "$1"; then vma="0"; else vma="$1"; fi
+if test -z "$2"; then vmi="1"; else vmi="$2"; fi
+if test -z "$3"; then vpl="0"; else vpl="$3"; fi
+if test -z "$4"; then vbld="0"; else vbld="$4"; fi
+
+AC_MSG_CHECKING(if chipcard_server support desired)
+AC_ARG_ENABLE(chipcard-server,
+  [  --enable-chipcard-server      enable chipcard_server support (default=yes)],
+  enable_chipcard_server="$enableval",
+  enable_chipcard_server="yes")
+AC_MSG_RESULT($enable_chipcard_server)
+
+have_chipcard_server="no"
+chipcard_server_dir=""
+chipcard_server_servicedir=""
+chipcard_server_driverdir=""
+chipcard_server_datadir=""
+if test "$enable_chipcard_server" != "no"; then
+  AC_MSG_CHECKING(for chipcard_server)
+  AC_ARG_WITH(chipcard-server-dir, [  --with-chipcard-server-dir=DIR
+                            uses chipcard_server from given dir],
+    [lcc_dir="$withval"],
+    [lcc_dir="${prefix} \
+	     /usr/local \
+             /usr \
+	     /chipcard-server \
+             /"])
+
+  for li in $lcc_dir; do
+      if test -x "$li/bin/chipcard3-config"; then
+          chipcard_server_dir="$li";
+          break
+      fi
+  done
+  if test -z "$chipcard_server_dir"; then
+      AC_MSG_RESULT([not found ])
+      AC_MSG_NOTICE([
+*** The library libchipcard3s was not found. Obtain it from 
+*** http://www.libchipcard.de.
+*** If it is already installed (including the -devel package), 
+*** you might need to specify the location with the 
+*** option --with-chipcard-server-dir=DIR.
+***
+*** Please note that it is not Libchipcard this package requested, it is the
+*** successor chipcard3.])
+  else
+      AC_MSG_RESULT($chipcard_server_dir)
+      AC_MSG_CHECKING(for chipcard-server datadir)
+      chipcard_server_datadir="`$chipcard_server_dir/bin/chipcard3-config --server-datadir`"
+      AC_MSG_RESULT($chipcard_server_datadir)
+      AC_MSG_CHECKING(for chipcard-server driver dir)
+      chipcard_server_driverdir="`$chipcard_server_dir/bin/chipcard3-config --driverdir`"
+      AC_MSG_RESULT($chipcard_server_driverdir)
+      AC_MSG_CHECKING(for chipcard-server service dir)
+      chipcard_server_servicedir="`$chipcard_server_dir/bin/chipcard3-config --servicedir`"
+      AC_MSG_RESULT($chipcard_server_servicedir)
+
+  fi
+  AC_MSG_CHECKING(if chipcard_server test desired)
+  AC_ARG_ENABLE(chipcard-server-test,
+    [  --enable-chipcard-server-test   enable chipcard_server-test (default=yes)],
+     enable_chipcard_server_test="$enableval",
+     enable_chipcard_server_test="yes")
+  AC_MSG_RESULT($enable_chipcard_server_test)
+  AC_MSG_CHECKING(for Chipcard-Server version >=$vma.$vmi.$vpl.$vbld)
+  if test "$enable_chipcard_server_test" != "no"; then
+    chipcard_server_versionstring="`$chipcard_server_dir/bin/chipcard3-config --vstring`.`$chipcard_server_dir/bin/chipcard3-config --vbuild`"
+    AC_MSG_RESULT([found $chipcard_server_versionstring])
+    if test "$vma" -gt "`$chipcard_server_dir/bin/chipcard3-config --vmajor`"; then
+      AC_MSG_NOTICE([Your Chipcard-Server version is way too old.
+      Please update from http://sf.net/projects/libchipcard])
+    elif test "$vma" = "`$chipcard_server_dir/bin/chipcard3-config --vmajor`"; then
+      if test "$vmi" -gt "`$chipcard_server_dir/bin/chipcard3-config --vminor`"; then
+        AC_MSG_NOTICE([Your Chipcard-Server version is too old.
+          Please update from http://sf.net/projects/libchipcard])
+      elif test "$vmi" = "`$chipcard_server_dir/bin/chipcard3-config --vminor`"; then
+          if test "$vpl" -gt "`$chipcard_server_dir/bin/chipcard3-config --vpatchlevel`"; then
+            AC_MSG_NOTICE([Your Chipcard-Server version is a little bit too old.
+            Please update from http://sf.net/projects/libchipcard])
+          elif test "$vpl" = "`$chipcard_server_dir/bin/chipcard3-config --vpatchlevel`"; then
+            if test "$vbld" -gt "`$chipcard_server_dir/bin/chipcard3-config --vbuild`"; then
+              AC_MSG_NOTICE([Your Chipcard-Server version is a little bit too old. 
+  Please update to the latest CVS version. Instructions for accessing 
+  CVS can be found on http://sf.net/projects/libchipcard])
+             fi
+           fi
+      fi
+    fi
+    have_chipcard_server="yes"
+    #AC_MSG_RESULT(yes)
+  else
+    have_chipcard_server="yes"
+    AC_MSG_RESULT(assuming yes)
+  fi
+dnl end of "if enable-chipcard-server"
+fi
+
+AC_SUBST(chipcard_server_servicedir)
+AC_SUBST(chipcard_server_driverdir)
+AC_SUBST(chipcard_server_datadir)
+])
+
+
--- origsrc/aqbanking-2.3.2/m4/kde3.m4	2006-04-08 03:03:15.000000000 -0500
+++ src/aqbanking-2.3.2/m4/kde3.m4	2007-07-16 07:12:08.796875000 -0500
@@ -158,7 +158,7 @@
 )
 
 if test -z "$kde3_libs"; then
-	AQ_SEARCH_FOR_PATH([libkdeui.so.4],[$local_kde3_libs])
+	AQ_SEARCH_FOR_PATH([libkdeui.dll.a],[$local_kde3_libs])
        	if test -n "$found_dir" ; then
        		kde3_libs="-L$found_dir"
        	fi
--- origsrc/aqbanking-2.3.2/m4/os.m4	2007-06-19 14:19:30.000000000 -0500
+++ src/aqbanking-2.3.2/m4/os.m4	2007-07-16 07:12:08.812500000 -0500
@@ -47,6 +47,11 @@
 	AC_DEFINE(OS_DARWIN,1,[if Darwin is used])
 	OS_TYPE="posix"
 	;;
+    *-cygwin*)
+	OSYSTEM="cygwin"
+	AC_DEFINE(OS_CYGWIN,1,[if Cygwin is used])
+	OS_TYPE="posix"
+	;;
     *-win32*)
     	OSYSTEM="windows"
 	AC_DEFINE(OS_WIN32,1,[if WIN32 is used])
--- origsrc/aqbanking-2.3.2/m4/qt3.m4	2006-12-29 11:07:40.000000000 -0600
+++ src/aqbanking-2.3.2/m4/qt3.m4	2007-07-16 07:12:08.843750000 -0500
@@ -149,11 +149,14 @@
 fi
 
 # This is the name of the qt library to search for.
-if test "x$OSYSTEM" = "xdarwin"; then
-   qt_searchname="lib${qt_libname}.3.dylib"
-else
-   qt_searchname="lib${qt_libname}${std_shrext}.3"
-fi
+case "$OSYSTEM" in
+darwin)
+   qt_searchname="lib${qt_libname}.3.dylib" ;;
+cygwin)
+   qt_searchname="lib${qt_libname}.dll.a" ;;
+*)
+   qt_searchname="lib${qt_libname}${std_shrext}.3" ;;
+esac
 
 
 if test -z "$qt3_libs"; then
@@ -235,6 +238,7 @@
 	AC_MSG_RESULT(not found)
 fi
 
+qt3_uic="$qt3_uic -nounload"
 
 # check if all necessary qt3 components where found
 if test -z "$qt3_includes" || \
--- origsrc/aqbanking-2.3.2/mksymlinks.sh	2007-07-08 05:29:22.000000000 -0500
+++ src/aqbanking-2.3.2/mksymlinks.sh	2007-07-17 01:56:48.000000000 -0500
@@ -78,13 +78,13 @@
 ln -s "${bprefix}/system.h" headers/aqbanking/system.h
 
 # symlink all headers from src/frontends
-for frontend in cbanking g2banking qbanking kbanking; do
+for frontend in cbanking g2banking qbanking kbanking fbanking; do
   mkdir -p headers/${frontend}
   symlinkFolder "src/frontends/${frontend}" "headers/${frontend}"
 done
 
 # symlink all headers from src/plugins/backends/
-for backend in  aqhbci aqdtaus aqgeldkarte aqofxconnect; do
+for backend in  aqhbci aqdtaus  aqofxconnect; do
   mkdir -p headers/${backend}
   if [ -d "src/plugins/backends/${backend}/plugin" ] ; then
     symlinkFolder "src/plugins/backends/${backend}/plugin" "headers/${backend}"
--- origsrc/aqbanking-2.3.2/src/frontends/cbanking/cbanking-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/src/frontends/cbanking/cbanking-config.in.in	2007-07-16 22:58:18.812500000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @cbanking_includes@"
+	    result="$result @cbanking_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @cbanking_libs@"
+	    result="$result @cbanking_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--data)
 	    result="$result @cbanking_data@"
--- origsrc/aqbanking-2.3.2/src/frontends/g2banking/g2banking-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/src/frontends/g2banking/g2banking-config.in.in	2007-07-17 00:33:58.453125000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @g2banking_includes@"
+	    result="$result @g2banking_includes@ @GTK2_CFLAGS@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @g2banking_libs@"
+	    result="$result @g2banking_libs@ @GTK2_LIBS@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--data)
 	    result="$result @g2banking_data@"
--- origsrc/aqbanking-2.3.2/src/frontends/kbanking/Makefile.am	2006-07-14 12:32:10.000000000 -0500
+++ src/aqbanking-2.3.2/src/frontends/kbanking/Makefile.am	2007-07-16 07:12:08.859375000 -0500
@@ -43,7 +43,7 @@
  views/libviews.la \
  ../qbanking/lib/libqbanking.la \
  $(aqbanking_internal_libs) \
- $(kde3_libs) -lkdeui -lkdecore -lkio $(gwenhywfar_libs)
+ $(kde3_libs) -lkdeui -lkdecore -lkio $(qt3_libs) $(gwenhywfar_libs)
 
 libkbanking_la_LDFLAGS = -no-undefined \
  -version-info @KBANKING_SO_CURRENT@:@KBANKING_SO_REVISION@:@KBANKING_SO_AGE@
--- origsrc/aqbanking-2.3.2/src/frontends/kbanking/kbanking-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/src/frontends/kbanking/kbanking-config.in.in	2007-07-17 01:53:39.015625000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @kbanking_includes@"
+	    result="$result @kbanking_includes@ @kde3_includes@ @qbanking_includes@ @qt3_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @kbanking_libs@"
+	    result="$result @kbanking_libs@ @kde3_libs@ -lkdeui -lkdecore @qbanking_libs@ @qt3_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--data)
 	    result="$result @kbanking_data@"
--- origsrc/aqbanking-2.3.2/src/frontends/qbanking/bin/helpbrowser/Makefile.am	2006-04-08 03:04:38.000000000 -0500
+++ src/aqbanking-2.3.2/src/frontends/qbanking/bin/helpbrowser/Makefile.am	2007-07-16 07:12:09.062500000 -0500
@@ -47,7 +47,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
--- origsrc/aqbanking-2.3.2/src/frontends/qbanking/lib/dialogs/Makefile.am	2006-04-08 03:04:34.000000000 -0500
+++ src/aqbanking-2.3.2/src/frontends/qbanking/lib/dialogs/Makefile.am	2007-07-16 07:12:09.062500000 -0500
@@ -132,7 +132,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/aqbanking-2.3.2/src/frontends/qbanking/lib/qbanking-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/src/frontends/qbanking/lib/qbanking-config.in.in	2007-07-16 23:02:39.812500000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @qbanking_includes@ @aqbanking_includes@ @gwenhywfar_includes@ @qt3_includes@"
+	    result="$result @qbanking_includes@ @qt3_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @qbanking_libs@"
+	    result="$result @qbanking_libs@ @qt3_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--plugins)
 	    result="$result @qbanking_plugindir@"
--- origsrc/aqbanking-2.3.2/src/frontends/qbanking/lib/views/Makefile.am	2006-04-08 03:04:35.000000000 -0500
+++ src/aqbanking-2.3.2/src/frontends/qbanking/lib/views/Makefile.am	2007-07-16 07:12:09.062500000 -0500
@@ -58,7 +58,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/aqbanking-2.3.2/src/frontends/qbanking/lib/widgets/Makefile.am	2006-04-08 03:04:37.000000000 -0500
+++ src/aqbanking-2.3.2/src/frontends/qbanking/lib/widgets/Makefile.am	2007-07-16 07:12:09.171875000 -0500
@@ -78,7 +78,7 @@
 	echo "#endif" >>$@ && \
 	cat $@.tmp >>$@ && \
 	rm $@.tmp && \
-	echo "#include \"$<.moc\"" >>$@
+	echo "#include \"$(@:.ui.cpp=.ui.moc)\"" >>$@
 
 
 %.ui.h: %.ui
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqdtaus/aqdtaus-config.in.in	2006-12-17 15:04:54.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqdtaus/aqdtaus-config.in.in	2007-07-16 23:03:49.796875000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @aqdtaus_includes@"
+	    result="$result @aqdtaus_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqdtaus_libs@"
+	    result="$result @aqdtaus_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--plugins)
 	    result="$result @aqdtaus_plugins@"
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqdtaus/ui/qt3/libs/Makefile.am	2006-04-08 03:03:58.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqdtaus/ui/qt3/libs/Makefile.am	2007-07-16 07:12:09.187500000 -0500
@@ -15,7 +15,7 @@
  cfgmoduledtaus_p.h
 
 
-aqdtaus_la_LDFLAGS=@STRIPALL@ -module -no-undefined
+aqdtaus_la_LDFLAGS=@STRIPALL@ -module -no-undefined -avoid-version
 
 aqdtaus_la_LIBADD=\
   dialogs/libdialogs.la \
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqdtaus/ui/qt3/libs/dialogs/Makefile.am	2006-04-08 03:03:58.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqdtaus/ui/qt3/libs/dialogs/Makefile.am	2007-07-16 07:12:09.187500000 -0500
@@ -37,7 +37,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqgeldkarte/aqgeldkarte-config.in.in	2006-12-17 15:04:54.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqgeldkarte/aqgeldkarte-config.in.in	2007-07-16 23:07:10.906250000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @aqgeldkarte_includes@"
+	    result="$result @aqgeldkarte_includes@ @chipcard_client_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqgeldkarte_libs@"
+	    result="$result @aqgeldkarte_libs@ @chipcard_client_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--plugins)
 	    result="$result @aqgeldkarte_plugins@"
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqgeldkarte/ui/qt3/libs/Makefile.am	2006-04-08 03:04:01.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqgeldkarte/ui/qt3/libs/Makefile.am	2007-07-16 07:12:09.406250000 -0500
@@ -15,7 +15,7 @@
  cfgmodulegeldkarte_p.h
 
 
-aqgeldkarte_la_LDFLAGS=@STRIPALL@ -module
+aqgeldkarte_la_LDFLAGS=@STRIPALL@ -module -avoid-version -no-undefined
 
 aqgeldkarte_la_LIBADD=\
   dialogs/libdialogs.la \
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/aqhbci-config.in.in	2006-12-17 15:04:54.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/aqhbci-config.in.in	2007-07-17 00:27:05.843750000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @aqhbci_includes@"
+	    result="$result @aqhbci_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqhbci_ldflags@ @aqhbci_libs@"
+	    result="$result @aqhbci_ldflags@ @aqhbci_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--plugins)
 	    result="$result @aqhbci_plugindir@"
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/plugin/Makefile.am	2007-02-13 14:31:25.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/plugin/Makefile.am	2007-07-16 07:12:09.406250000 -0500
@@ -88,7 +88,7 @@
 
 testlib_SOURCES=testlib.c
 testlib_LDADD = \
-  $(aqhbci_internal_libs) \
+  libaqhbci.la \
   $(cbanking_internal_libs) \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/Makefile.am	2006-04-08 03:04:16.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/Makefile.am	2007-07-16 07:12:09.421875000 -0500
@@ -16,7 +16,7 @@
  cfgmodulehbci_p.h
 
 
-aqhbci_la_LDFLAGS=@STRIPALL@ -module -no-undefined
+aqhbci_la_LDFLAGS=@STRIPALL@ -module -no-undefined -avoid-version
 
 aqhbci_la_LIBADD=\
   dialogs/libdialogs.la \
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/Makefile.am	2006-04-08 03:04:13.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/Makefile.am	2007-07-16 07:12:09.562500000 -0500
@@ -62,7 +62,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/wizard/Makefile.am	2006-04-08 03:04:12.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/dialogs/wizard/Makefile.am	2007-07-16 07:12:09.562500000 -0500
@@ -115,7 +115,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am	2006-04-08 03:04:14.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am	2007-07-16 07:12:09.796875000 -0500
@@ -52,7 +52,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqofxconnect/aqofxconnect-config.in.in	2006-12-17 15:04:55.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqofxconnect/aqofxconnect-config.in.in	2007-07-17 00:27:16.625000000 -0500
@@ -12,10 +12,10 @@
 for d in $*; do
     case $d in
 	--includes)
-	    result="$result @all_includes@ @aqofxconnect_includes@"
+	    result="$result @aqofxconnect_includes@ @aqbanking_includes@ @gwenhywfar_includes@"
 	    ;;
 	--libraries)
-	    result="$result @aqofxconnect_libs@"
+	    result="$result @aqofxconnect_libs@ @aqbanking_libs@ @gwenhywfar_libs@"
 	    ;;
 	--plugins)
 	    result="$result @aqofxconnect_plugins@"
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqofxconnect/ui/qt3/libs/Makefile.am	2007-02-13 12:23:10.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqofxconnect/ui/qt3/libs/Makefile.am	2007-07-16 07:12:09.812500000 -0500
@@ -33,7 +33,7 @@
  cfgtabpageaccountofx.ui.h \
  cfgtabpageuserofx.ui.h
 
-aqofxconnect_la_LDFLAGS=@STRIPALL@ -module
+aqofxconnect_la_LDFLAGS=@STRIPALL@ -module -avoid-version -no-undefined
 
 aqofxconnect_la_LIBADD=\
   $(aqofxconnect_internal_libs) \
@@ -55,7 +55,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqyellownet/plugin/Makefile.am	2007-01-12 15:30:06.000000000 -0600
+++ src/aqbanking-2.3.2/src/plugins/backends/aqyellownet/plugin/Makefile.am	2007-07-16 07:12:09.828125000 -0500
@@ -30,7 +30,7 @@
   -L$(srcdir) -laqyellownet \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
-aqyellownet_la_LDFLAGS=-module -avoid-version
+aqyellownet_la_LDFLAGS=-module -avoid-version -no-undefined
 
 #  -L$(srcdir) -laqyellownet
 #  $(srcdir)/libaqyellownet.la
--- origsrc/aqbanking-2.3.2/src/plugins/backends/aqyellownet/ui/qt3/libs/Makefile.am	2006-04-08 03:04:03.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/backends/aqyellownet/ui/qt3/libs/Makefile.am	2007-07-16 07:12:09.906250000 -0500
@@ -32,7 +32,7 @@
  cfgtabpageuseryn.cpp
 
 
-aqyellownet_la_LDFLAGS=@STRIPALL@ -module
+aqyellownet_la_LDFLAGS=@STRIPALL@ -module -avoid-version -no-undefined
 
 aqyellownet_la_LIBADD=\
   $(aqyellownet_internal_libs) \
@@ -58,7 +58,7 @@
 
 # Build rules for the files that come from the .ui file
 %.ui.cpp: %.ui
-	$(UIC3) $< -impl $<.h -o $@
+	$(UIC3) $< -impl $(@:.ui.cpp=.ui.h) -o $@
 %.ui.h: %.ui
 	$(UIC3) $< -o $@
 SUFFIXES = .ui .ui.h .ui.cpp .moc.cpp
--- origsrc/aqbanking-2.3.2/src/plugins/bankinfo/at/Makefile.am	2006-04-08 03:03:27.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/bankinfo/at/Makefile.am	2007-07-16 07:12:09.921875000 -0500
@@ -13,7 +13,7 @@
  at.c
 
 at_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs)
-at_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+at_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/aqbanking-2.3.2/src/plugins/bankinfo/ch/Makefile.am	2006-04-08 03:03:28.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/bankinfo/ch/Makefile.am	2007-07-16 07:12:09.921875000 -0500
@@ -13,7 +13,7 @@
  ch.c
 
 ch_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs)
-ch_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+ch_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/aqbanking-2.3.2/src/plugins/bankinfo/de/Makefile.am	2006-04-08 03:03:28.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/bankinfo/de/Makefile.am	2007-07-16 07:12:10.234375000 -0500
@@ -13,7 +13,7 @@
  de.c
 
 de_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la @ktoblzcheck_libs@ $(gwenhywfar_libs)
-de_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+de_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/aqbanking-2.3.2/src/plugins/bankinfo/us/Makefile.am	2006-04-08 03:03:27.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/bankinfo/us/Makefile.am	2007-07-16 07:12:10.250000000 -0500
@@ -13,7 +13,7 @@
  us.c
 
 us_la_LIBADD= $(aqbanking_internal_libs) ../generic/libbankinfo.la $(gwenhywfar_libs)
-us_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+us_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 
 sources:
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/csv/Makefile.am	2006-04-08 03:03:25.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/csv/Makefile.am	2007-07-16 07:12:10.265625000 -0500
@@ -22,7 +22,7 @@
  csv.c
 
 csv_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-csv_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+csv_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 csv_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/dtaus/Makefile.am	2006-04-08 03:03:22.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/dtaus/Makefile.am	2007-07-16 07:12:10.515625000 -0500
@@ -19,7 +19,7 @@
  dtaus.c
 
 dtaus_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-dtaus_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+dtaus_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 dtaus_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/eri/Makefile.am	2006-04-08 03:03:21.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/eri/Makefile.am	2007-07-16 07:12:10.531250000 -0500
@@ -21,7 +21,7 @@
  eri.c
 
 eri_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-eri_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+eri_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 eri_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/eri2/Makefile.am	2006-04-08 03:03:23.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/eri2/Makefile.am	2007-07-16 07:12:10.531250000 -0500
@@ -22,7 +22,7 @@
  eri2.c
 
 eri2_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-eri2_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+eri2_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 eri2_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/ofx/Makefile.am	2006-04-08 03:03:25.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/ofx/Makefile.am	2007-07-16 07:12:10.687500000 -0500
@@ -25,7 +25,7 @@
 
 ofx_la_LIBADD=$(aqbanking_internal_libs) -lofx $(gwenhywfar_libs) 
 ofx_la_LDFLAGS = -no-undefined @STRIPALL@ \
-  -module -omit-version
+  -module -avoid-version
 
 # Add the linker flags for windows resource file
 ofx_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/openhbci1/Makefile.am	2006-04-08 03:03:24.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/openhbci1/Makefile.am	2007-07-16 07:12:10.703125000 -0500
@@ -19,7 +19,7 @@
  openhbci1.c
 
 openhbci1_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-openhbci1_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+openhbci1_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 openhbci1_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/qif/Makefile.am	2006-04-08 03:03:22.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/qif/Makefile.am	2007-07-16 07:12:10.703125000 -0500
@@ -19,7 +19,7 @@
  qif.c
 
 qif_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs)
-qif_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+qif_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 qif_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/swift/Makefile.am	2006-04-08 03:03:26.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/swift/Makefile.am	2007-07-16 07:12:10.921875000 -0500
@@ -20,7 +20,7 @@
  swift.c
 
 swift_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-swift_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+swift_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 swift_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/xmldb/Makefile.am	2006-06-23 13:12:31.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/xmldb/Makefile.am	2007-07-16 07:12:10.937500000 -0500
@@ -20,7 +20,7 @@
  xmldb.c
 
 xmldb_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-xmldb_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+xmldb_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 xmldb_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/imexporters/yellownet/Makefile.am	2006-06-16 12:05:58.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/imexporters/yellownet/Makefile.am	2007-07-16 07:12:10.968750000 -0500
@@ -19,7 +19,7 @@
  yellownet.c
 
 yellownet_la_LIBADD=$(aqbanking_internal_libs) $(gwenhywfar_libs) 
-yellownet_la_LDFLAGS = -no-undefined @STRIPALL@ -module -omit-version
+yellownet_la_LDFLAGS = -no-undefined @STRIPALL@ -module -avoid-version
 
 # Add the linker flags for windows resource file
 yellownet_la_DEPENDENCIES = \
--- origsrc/aqbanking-2.3.2/src/plugins/parsers/dtaus/Makefile.am	2006-04-08 03:03:19.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/parsers/dtaus/Makefile.am	2007-07-16 07:12:11.625000000 -0500
@@ -25,7 +25,7 @@
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
 
-dtaus_la_LDFLAGS = -no-undefined $(STRIPALL) -module -omit-version
+dtaus_la_LDFLAGS = -no-undefined $(STRIPALL) -module -avoid-version
 
 
 # Add the linker flags for windows resource file
--- origsrc/aqbanking-2.3.2/src/plugins/parsers/swift/Makefile.am	2006-04-08 03:03:20.000000000 -0500
+++ src/aqbanking-2.3.2/src/plugins/parsers/swift/Makefile.am	2007-07-16 07:12:12.000000000 -0500
@@ -24,7 +24,7 @@
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
 
-swift_la_LDFLAGS = -no-undefined $(STRIPALL) -module -omit-version
+swift_la_LDFLAGS = -no-undefined $(STRIPALL) -module -avoid-version
 
 
 # Add the linker flags for windows resource file
